tpot/tpot.py

```diff
@@ -1657,13 +1657,16 @@ def _combined_selection_operator(self, individuals, k):
         """
         return tools.selNSGA2(individuals, int(k / 5.)) * 5
 
-    def _random_mutation_operator(self, individual):
+    def _random_mutation_operator(self, individual, mutation_type=None):
```

Comment:
Yeah, but the alternative felt a bit janky - in that I'd need to find 3 separate seeds that would lead me to each branch of the method